synapseclient/annotations.py

```diff
@@ -294,17 +294,18 @@ def __init__(
         Create an Annotations object taking key value pairs from a dictionary or from keyword arguments.
         System properties id, etag, creationDate and uri become attributes of the object.
 
-        :param id:  A Synapse ID, a Synapse Entity object, a plain dictionary in which 'id' maps to a Synapse ID
-        :param etag: etag of the Synapse Entity
-        :param values:  (Optional) dictionary of values to be copied into annotations
-        :param \**kwargs: additional key-value pairs to be added as annotations
-
-        Example::
-
-            example1 = Annotations('syn123','40256475-6fb3-11ea-bb0a-9cb6d0d8d984', {'foo':'bar'})
-            example2 = Annotations('syn123','40256475-6fb3-11ea-bb0a-9cb6d0d8d984', foo='bar')
-            example3 = Annotations('syn123','40256475-6fb3-11ea-bb0a-9cb6d0d8d984')
-            example3['foo'] = 'bar'
+        Attributes:
+            id:  A Synapse ID, a Synapse Entity object, a plain dictionary in which 'id' maps to a Synapse ID
+            etag: etag of the Synapse Entity
+            values: (Optional) dictionary of values to be copied into annotations
+            **kwargs: additional key-value pairs to be added as annotations
+
+        Example: Creating a few instances
+            Creating and setting annotations
+                example1 = Annotations('syn123','40256475-6fb3-11ea-bb0a-9cb6d0d8d984', {'foo':'bar'})
```

Comment:
While we are touching this let's add to this and include the import as well to the example:
```
from synapseclient import Annotations
```